AutoscoperM/AutoscoperM.py

```diff
@@ -651,6 +651,14 @@ def onLoadPV(self):
         vols = glob.glob(os.path.join(mainOutputDir, volumeSubDir, "*.tif"))
         tfms = glob.glob(os.path.join(mainOutputDir, transformSubDir, "*.tfm"))
 
+        if len(vols) != len(tfms):
```

Comment:
I restored the original check, while pairing make sense, I think we should expect the same number of transform and volume ?

Reply:
We are writing other transforms for the `Transforms` directory, so the number of tfm files will be higher than the number of volumes.

Reply:
Thanks for clarifying, I removed the extra check and only kept the one ensuring there is at least one volume.